spec/yaml/swiftnav/sbp/navigation.yaml

```diff
@@ -700,7 +704,10 @@ definitions:
             desc: Status flags
             fields:
               - 6-7:
-                  desc: Reserved
+                  desc: Geoid model
```

Comment:
Are there more models to add in here in the future? Otherwise bit 7 might not be needed

Reply:
Yup, the geoid will be updates as more or better survey data becomes available, EGM2020 seems to be the upcoming release: https://ui.adsabs.harvard.edu/abs/2015AGUFM.G34A..03B/abstract

Reply:
The upcoming EGM2020 may take up one of the two remaining slots, leaving only one more. Just wondering if this is enough. We could try to rename and reuse part of the `confidence` field, if we want more options.

Reply:
Yeah, better use 3 bits for the model.

Reply:
Ok. I'll move this to the `confidence` field, and include a "invalid geoid" option as well. I don't think we're likely to add a model that has invalid regions, but it's safer to leave that possibility open.